omniscidb/ArrowStorage/ArrowStorage.h

```diff
@@ -85,6 +85,13 @@ class ArrowStorage : public SimpleSchemaProvider, public AbstractDataProvider {
   TableInfoPtr importArrowTable(std::shared_ptr<arrow::Table> at,
                                 const std::string& table_name,
                                 const TableOptions& options = TableOptions());
+  TableInfoPtr getRefragmentedView(const std::string& table_name,
+                                   const std::string& new_table_name,
+                                   const size_t new_frag_size);
+  TableInfoPtr createRefragmentedView(const std::string& table_name,
+                                      const std::string& new_table_name,
+                                      const size_t new_frag_size);
+  void refragmentTable(const std::string& table_name, const size_t new_frag_size);
```

Comment:
Do we need this methods in public? As far as I see `getRefragmentedView` uses both this  methods, maybe only this method is enough?